### PR TITLE
[DUOS-2294][DUOS-2295][risk=no] Update notifications for minimized navigation panel

### PR DESCRIPTION
**Pull request by rushtong**
## Addresses
https://broadworkbench.atlassian.net/browse/DUOS-2294
https://broadworkbench.atlassian.net/browse/DUOS-2295

This PR updates the minimized drawer navigation panel so that it doesn't try to render the notifications a second time. It also reduces the spacing over the notification. Lastly, it addresses some react console warnings about missing keys in the navigation menu.

### New
Screenshot 2023-02-06 at 10 02 21 AM

### Old
Screenshot 2023-02-03 at 3 02 29 PM

----
Have you read [Terra's Contributing Guide](https://github.com/DataBiosphere/terra-ui/wiki/Contributor-Guide) lately? If not, do that first.

- Label PR with a Jira ticket number and include a link to the ticket
- Label PR with a security risk modifier [no, low, medium, high]
- PR describes scope of changes
- Get a minimum of one thumbs worth of review, preferably two if enough team members are available
- Get PO sign-off for all non-trivial UI or workflow changes
- Verify all tests go green
- Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: src/pages/DMSPolicyInfo.css [hunk 1]
```diff
@@ -126,10 +126,6 @@
   margin-bottom: auto !important;
 }
 
-p {
-  margin-top: 10px;
-}
```
Comment (rushtong) [inline src/pages/DMSPolicyInfo.css]: This global style update interfered with other `p` tags across the site.

**Review by fboulnois — APPROVED** [review]
looks great!

File: src/pages/DMSPolicyInfo.css (hunk 1 above)
Comment (nmalfroy) [reply src/pages/DMSPolicyInfo.css]: Gah sorry about that. Rather than deleted, is it possible to scope it?  I think this might mess up the DMS page

Comment (rushtong) [reply src/pages/DMSPolicyInfo.css]: I think we have to scope it on the DMS page. The `p` tags where I'm seeing issues is when we convert text to markdown in the notifications. Let me take another stab at it ... might be able to make something work.

Comment (nmalfroy) [reply src/pages/DMSPolicyInfo.css]: I think you should be able to just do:
```
.base-style p {
  margin-top: 10px;
}
```

(probably need a more dms-page specific base class)

I also noticed the same thing right below for the `b` tag

Comment (rushtong) [reply src/pages/DMSPolicyInfo.css]: OK, I think I have a fix for it - just have to override what's there.

Comment (rushtong) [reply src/pages/DMSPolicyInfo.css]: I ended up reverting this change so we can address both global styles separately

Comment (nmalfroy) [reply src/pages/DMSPolicyInfo.css]: Ah gotcha

**Review by nmalfroy — APPROVED** [review]
looks good!

**Pull request merged**